packages/design-system-tokens/src/tokens/font.ts

```diff
@@ -8,39 +8,29 @@ const makeFonts = <T extends FontTokens>(value: T) => {
 };
 
 const font = makeFonts({
-  family: {
-    'open-sans': '"Open Sans", Helvetica, sans-serif',
-    rubik: '"Rubik", sans-serif',
-    montserrat: '"Montserrat", sans-serif',
-    bitter: 'Bitter, Georgia, serif',
-  },
-  size: {
-    base: '16px',
-    sm: '14px',
-    md: '16px',
-    lg: '18px',
-    xl: '21px',
-    '2xl': '24px',
-    '3xl': '36px',
-    '4xl': '48px',
-    '5xl': '60px',
-  },
-  lineHeight: {
-    reset: '1',
-    base: '1.5',
-    heading: '1.3',
-    lead: '1.7',
-  },
-  weight: {
-    normal: '400',
-    bold: '700',
-    semibold: '600',
-  },
-  measure: {
-    narrow: '45ex',
-    base: '65ex',
-    wide: '80ex',
-  },
+  'family-open-sans': '"Open Sans", Helvetica, sans-serif',
+  'family-rubik': '"Rubik", sans-serif',
+  'family-montserrat': '"Montserrat", sans-serif',
+  'family-bitter': 'Bitter, Georgia, serif',
+  'size-base': '16px',
+  'size-sm': '14px',
```

Comment:
Okay, yeah, in the final generated tokens it was just one string anyway, right? I think that's a good simplification